R/map_adjust_dose.R

```diff
@@ -0,0 +1,76 @@
+#' Adjust doses to achieve a target metric using MAP Bayesian estimation.
```

Comment:
map_adjust_dose and map_adjust_interval were just separated off as their own files, for convenience. No code changes